src/parser.js

```diff
@@ -530,41 +538,27 @@ function mapIfStatement(node, meta) {
   );
 }
 
-function isStatement(expr) {
-  const type = expr.constructor.name;
-  switch (type) {
-    case 'Literal':
-      if (expr.value === 'break' || expr.value === 'continue') {
-        return true;
-      }
-      return false;
-    case 'Throw':
-    case 'For':
-    case 'While':
-    case 'Return':
-    case 'If':
-    case 'Break':
-      return true;
-    default:
-      return false;
-  }
-}
+function isTernaryOperation(node, isExpressionCondition) {
```

Comment:
Manual change to get rid of IIFEs for assignments ([mainline code](https://github.com/juliankrispel/decaf/blob/master/src/parser.js#L521)).

Mainline was generating stuff like:
```js
 return $(this).addClass((() => {
   if (!$(this).hasClass('disabled')) {
     return 'is-active';
   }
 })());
```

This tweak results gets us back to:
```js
  return $(this).addClass((!$(this).hasClass('disabled') ? 'is-active' : undefined));
```

I dislike having to pass around this parameter like this.  Maybe it'd be easier to just keep the IIFEs and transform them out?